app/assets/src/components/common/DetailsSidebar/TaxonDetailsMode/TaxonDetailsMode.jsx

```diff
@@ -259,7 +277,15 @@ export default class TaxonDetailsMode extends React.Component {
               this.state.parentDescriptionTall && (
                 <div
                   className={cs.expandLink}
-                  onClick={this.expandParentDescription}
+                  onClick={withAnalytics(
+                    this.expandParentDescription,
+                    "TaxonDetailsMode_show-more-link_clicked",
```

Comment:
Should distinguish this from the one above. Maybe`TaxonDetailsMode_genus-show-more-link_clicked`?

Reply:
good idea!